subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/OutputFileProvider.kt

```diff
@@ -0,0 +1,36 @@
+package com.avito.android.test.report.transport
+
+import androidx.test.platform.app.InstrumentationRegistry
+import java.io.File
+
+internal class OutputFileProvider {
```

Comment:
It's a too abstract name for the class which provides a `ReportFile`.

I want consistency. You could rename class or function ie:
- FileProvider and get(): Result<File>
- ReportProvider and get(): Result<Report>

Reply:
yep, i agree; i'll add this comment to Report extraction story